### ISSUE TITLE
Matomo Analytics opt-out sucess message not announced to screen readers

### ISSUE THREAD

**Issue by amberhinds** [wpa11yday/wpaccessibilityday#93]
See: https://2024.wpaccessibility.day/dont-track-me/

After unchecking the box, the label changes and a paragraph appears below it confirming the opt-out. It is not announced to screen reader users.

<img width="1042" alt="Screenshot 2024-02-10 at 12 29 19 PM" src="https://github.com/wpa11yday/wpaccessibilityday/assets/13153456/80e51161-0796-47f8-9aa2-26bcc9976b98">

Can we add aria-live to this so it's announced?
